newsfragments/2983.internal.rst

```diff
@@ -0,0 +1 @@
+Moved `ethpm` deprecation warning to only show when the module is explicitly enabled
```

Comment:
Just double checking... wrapping text in single backticks ` will [italicize](https://web3py.readthedocs.io/en/latest/releases.html#deprecations) in ReadTheDocs, wrapping in double backticks `` will create [code text](https://web3py.readthedocs.io/en/latest/releases.html#features). I think either could make sense here but we seem to have varying use of double vs single in the docs. Whatever you mean to do here, do it 😆.

Reply:
Thanks! This PR now includes updates to a few other newsfragments.